syndeo-template/README.md

```diff
@@ -73,20 +73,22 @@ Supported `SchemaTransform` and configuration parameters are configured in `Synd
 
 ### Run tests (unit and integration tests)
 
-Note that syndeo depends of the teleport integration testing framework, so make sure to install that locally:
+Note that syndeo depends on the teleport integration testing framework, so make sure to install that locally:
 
-```
-mvn install -DskipTests -f pom.xml -pl .,it/pom.xml
+```shell
+mvn install -DskipTests -f pom.xml -pl .,metadata,it/pom.xml
```

Comment:
Why not `-pl syndeo-template -am` instead of requiring a separate step to install deps?

Reply:
Neat! I wasn't aware of this way of building it!